src/components/Documents/DocumentTableRow.jsx

```diff
@@ -66,8 +66,8 @@ const DocumentTableRow = ({ document, handlePermissions }) => {
         </IconButton>
       </StyledTableCell>
       <StyledTableCell align="center">
-        <IconButton type="button" onClick={() => handlePermissions(name)}>
-          <SettingsIcon />
+        <IconButton type="button" onClick={() => handleModal('document', name)}>
+          <ShieldIcon />
```

Comment:
Similar to the case with the Shield icon in the set permission button, but I think the previous iteration with the gear icon looks better in this case.